### PR TITLE
validate skip.json lines 500-600

### PR DESCRIPTION
**Pull request by jfortin-sap**
**What this PR does / why we need it**:
Validated 1.20 skip.json lines 500-600, purged no longer found in skip.json and moved working ones to working.json.
**Which issue(s) this PR fixes**:
Fixes #416

**Special notes for your reviewer**:

**Release note**:

```improvement operator
Validated 1.20 skip.json lines 500-600, purged no longer found in skip.json and moved working ones to working.json.
```

### CODE REVIEW

**Review by dguendisch — APPROVED** [review]
/lgtm

**Comment by jfortin-sap** [comment]
@neo-liang-sap Can you merge this? I don't got permissions

**Comment by neo-liang-sap** [comment]
I don't have write permission to this repo either, @dguendisch @schrodit could you please merge it? thanks!

File: integration-tests/e2e/kubetest/description/1.20/working.json [hunk 1]
```diff
@@ -475,5 +475,28 @@
   { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] subPath should support non-existent path", "groups": ["fast"]},
   { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] subPath should support readOnly directory specified in the volumeMount", "groups": ["fast"]},
   { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] subPath should support restarting containers using directory as subpath [Slow]", "groups": ["fast"]},
-  { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] volumes should allow exec of files on the volume", "groups": ["fast"]}
+  { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Dynamic PV (default fs)] volumes should allow exec of files on the volume", "groups": ["fast"]},
+  { "testcase": "[sig-storage] In-tree Volumes [Driver: emptydir] [Testpattern: Inline-volume (default fs)] subPath should verify container cannot write to subpath readonly volumes [Slow]"},
```
Comment (schrodit) [inline integration-tests/e2e/kubetest/description/1.20/working.json:479]: there are lot of testcases without a group assigned. 
@dguendisch do you know if we have a test definition for such cases or do they never get executed?

**Review by schrodit — CHANGES_REQUESTED** [review]

**Comment by jfortin-sap** [comment]
@dguendisch Ready to merge?

**Comment by jfortin-sap** [comment]
@schrodit Ready to merge?

**Review by schrodit — APPROVED** [review]
/lgtm

**Comment by schrodit** [comment]
@jfortin-sap looks good but you have to rebase then I will merge the PR

**Comment by jfortin-sap** [comment]
@schrodit Re-based

**Comment by schrodit** [comment]
@jfortin-sap there seems something wrong with your rebase. Now there are ~300 changed files.
Can do you do a `git rebase master` so that only your changes are included in the PR? (Otherwise its hard to judge that it will not overwrite something important)

**Pull request closed without merging**